meta/config.go

```diff
@@ -37,10 +37,18 @@ type Config struct {
 	HeartbeatTimeout    toml.Duration `toml:"heartbeat-timeout"`
 	LeaderLeaseTimeout  toml.Duration `toml:"leader-lease-timeout"`
 	CommitTimeout       toml.Duration `toml:"commit-timeout"`
+	ClusterTracing      bool          `toml:"cluster-tracing"`
+
+	// The join command-line argument
+	join string
```

Comment:
It's set as a command-line arg.  It it's public, `influxd config` lists it as a config option which is not valid.